src/xdocs/google_style.xml

```diff
@@ -2267,12 +2287,15 @@
                         alt="" />
                   </span>
                   <a href="checks/javadoc/missingjavadocmethod.html#MissingJavadocMethod">MissingJavadocMethod</a>
+                  <br />
+                  <br />
                   <span class="wrapper inline">
                     <img
                         src="images/ok_blue.png"
                         alt="" />
                   </span>
-                  <a href="checks/javadoc/javadocmethod.html#JavadocMethod">JavadocMethod</a><br/>
+                  <a href="checks/javadoc/javadocmethod.html#JavadocMethod">JavadocMethod</a>
+                  <br />
                   Overrides are checked by presence of "@Override" annotation on
```

Comment:
This description belongs to `MissingJavadocMethod`, could you position it just below it, otherwise it looks like it belongs to `JavadocMethod`.

Reply:
done